track_and_trace/server/system/submit_utils.js

```diff
@@ -19,23 +19,30 @@
 const _ = require('lodash')
 const request = require('request-promise-native')
 const { createHash } = require('crypto')
-const secp256k1 = require('sawtooth-sdk/signing/secp256k1')
+const { createContext, CryptoFactory } = require('sawtooth-sdk/signing')
 const {
   Transaction,
   TransactionHeader,
-  TransactionList
+  Batch,
+  BatchHeader,
+  BatchList
 } = require('sawtooth-sdk/protobuf')
 const protos = require('../blockchain/protos')
 
 const FAMILY_NAME = 'grid_track_and_trace'
 const FAMILY_VERSION = '1.0'
-const NAMESPACE = 'a43b46'
 
-const SERVER = process.env.SERVER || 'http://localhost:3000'
+const SERVER = 'http://tnt-asset-client:8021'
```

Comment:
This shouldn't be locked to this URL, as it is very specific to the docker compose file.  It will break if used in any other context.

Reply:
Updated